test/README.md

```diff
@@ -77,11 +77,10 @@ To run all legacy Vim tests:
 
     make oldtest
 
-To run a *single* legacy test set `TEST_FILE`, for example:
+To run a *single* legacy test file:
 
-    TEST_FILE=test_syntax.res make oldtest
+    make src/nvim/testdir/test_syntax.vim
```

Comment:
is TEST_FILE not supported now? It makes more sense  to recommend that if it still works, otherwise people have to remember different ways of running tests for Lua vs oldtest.

Reply:
ok, will change that to TEST_FILE=test_syntax.vim then.
Maybe we can mention both though, if you would like to have support for the full name.

Reply:
@justinmk 
Should I mention both here?

Reply:
> Should I mention both here?

If the plan is to make it work for Lua tests also, then I guess that makes sense.  In general though too many options makes docs hard to read.

Reply:
ok, amended.